bundles/framework/admin-layeranalytics/Tile.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * @class Oskari.framework.bundle.admin-layeranalytics.Tile
+ * Renders the "layer analytics admin" tile.
+ */
+Oskari.clazz.define('Oskari.framework.bundle.admin-layeranalytics.Tile',
```

Comment:
Nothing particularly wrong here but it's a lot of code just to get a "tile" without any customization. We have a DefaultTile under framework/divmanazer/extension that could help with this or you could extend DefaultExtension with instance.js. That would make this file unnecessary. It's ok for now but I really wish we could do this much simpler.